apps/rush-lib/src/logic/Autoinstaller.ts

```diff
@@ -80,13 +80,17 @@ export class Autoinstaller {
 
     console.log();
 
+    Utilities.syncNpmrc(this._rushConfiguration.commonRushConfigFolder, this.folderFullPath);
+
     Utilities.executeCommand({
       command: this._rushConfiguration.packageManagerToolFilename,
       args: ['install'],
       workingDirectory: this.folderFullPath,
       keepEnvironment: true
     });
 
+    FileSystem.deleteFile(path.join(this.folderFullPath, '/.npmrc'));
```

Comment:
Why delete after? Given that we sync the most current NPMRC (or delete if the current NPMRC isn't found) before running the command, would the delete be necessary?

Reply:
I was thinking if rush should preserve user's file structure, but I just found rush project template ignored the file. So I will remove this.

Reply:
Done.